lib/next_rails/bundle_report.rb

```diff
@@ -2,6 +2,7 @@
 require "cgi"
 require "erb"
 require "json"
+require "rails_ruby_version_map"
```

Comment:
I know that I said to store this information somewhere when I created the feature request, but now I'm thinking this can become hard to maintain, maybe it's possible to get the data from rubygems on the fly?

Reply:
I have checked and there is an API that rubygems provide to fetch info about any gem and version

this is the api 
`GET https://rubygems.org/api/v2/rubygems/rails/versions/7.0.0.json`

and it gives a json response with key - value as `"ruby_version": ">= 2.7.0"`
I should be able to generate this data.